doc/docs/Parallel_Meep.md

```diff
@@ -36,7 +36,7 @@ mpirun -np 4 meep foo.ctl > foo.out
 
 There is one important requirement: every MPI process must be able to read the `foo.py`/`foo.ctl` input file or whatever your script file is called. On most systems, this is no problem, but if for some reason your MPI processes don't all have access to the local filesystem then you may need to make copies of your input file or something. This requirement also applies to HDF5 files used for input (i.e., via `epsilon_input_file`) or output (i.e., `output_epsilon()`, `output_efield()`, etc.). Any disruptions to the network or disk failures on individual machines which affect the [network file system](https://en.wikipedia.org/wiki/Network_File_System) may cause Meep to freeze/hang.
 
-For a potential improvement in [load balancing](FAQ.md#should-i-expect-linear-speedup-from-the-parallel-meep), you can try setting [`split_chunks_evenly=False`](Python_User_Interface.md#the-simulation-class) in the `Simulation` constructor. For a technical description of the load balancing features in Meep as well as some performance metrics, see [arXiv:2003.04287](https://arxiv.org/abs/2003.04287).
+For a potential improvement in [load balancing](FAQ.md#should-i-expect-linear-speedup-from-the-parallel-meep), you can try setting [`split_chunks_evenly=False`](Python_User_Interface.md#the-simulation-class) in the `Simulation` constructor. When there are a large number of chunks, this may also require *reducing* the default values for `subpixel_maxeval=100000` and `subpixel_tol=0.0001` to e.g. `10000` and `0.01`, respectively, to speed up the cell-partitioning algorithm (which otherwise can take several *hours* for a large heterogeneous 3d cell with more than ~100 chunks). These two parameters are used internally by the cell-partitioning algorithm as part of an adaptive quadrature scheme to compute the volume overlap of arbitrary subregions with the geometric objects over and over again when exploring a large space of possible cell partitions. For a technical description of the load-balancing features in Meep as well as some performance metrics from actual experiments, see [arXiv:2003.04287](https://arxiv.org/abs/2003.04287).
```

Comment:
I think we should just hard-code lower tolerances when we do the integrals for the fragment statistics.  e.g. set them to 1000 and 0.05

Reply:
Done.